app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt

```diff
@@ -15,6 +15,17 @@ import io.github.droidkaigi.confsched2018.model.toReadableDateTimeString
 import io.github.droidkaigi.confsched2018.model.toReadableTimeString
 import java.util.regex.Pattern
 
+@BindingAdapter(value = ["bind:dayNumber", "bind:suffix"])
+fun TextView.setDayText(dayNumber: Int?, suffix: String?) {
```

Comment:
These two args should be non-null, right?
If so, can you remove `?`?

Reply:
Indeed.
I'll fix that. Thanks!